Makefile

```diff
@@ -71,12 +71,25 @@ build-disk: build-os
 	mkdir -p $(ROOT_DIR)/build
 	$(DOCKER) run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $(ROOT_DIR)/build:/build \
 		--entrypoint /usr/bin/elemental \
-		${TOOLKIT_REPO}:${VERSION} --debug build-disk --unprivileged --expandable -n elemental-$(FLAVOR).$(ARCH) --local \
+		${TOOLKIT_REPO}:${VERSION} --debug build-disk --platform $(PLATFORM) --unprivileged --expandable -n elemental-$(FLAVOR).$(ARCH) --local \
```

Comment:
Uhm, we already have TARGETOS and TARGETARCH - and PLATFORM seems to be a combination of both ?!

Reply:
Hehe, TARGETOS and TARGETARCH are set in Dockerfile based on what platform is passed in.. Generally what we use is `--platform` but the fun begins when we are doing cross-compilation in Dockerfiles :see_no_evil: